src/main/java/io/algopreorg/alog/containsduplicate/Solution.java

```diff
@@ -1,8 +1,17 @@
 package io.algopreorg.alog.containsduplicate;
 
+import java.util.HashSet;
+import java.util.Set;
+
 public class Solution {
 
     public boolean containsDuplicate(int[] nums) {
+        Set<Integer> integerSet = new HashSet<>();
```

Comment:
It will be good to add more useful names for instance uniqueNumbers

Reply:
Thanks, fixed)